src/systems/ec/implementations/vanilla/packages/ec-component-timeline2/ec-component-timeline2.js

```diff
@@ -0,0 +1,91 @@
+import { queryOne } from '@ecl/ec-base/helpers/dom';
+
+export class Timeline2 {
+  constructor(
+    element,
+    {
+      buttonSelector: buttonSelector = '[data-ecl-timeline-button]',
```

Comment:
we should have renamed it `data-ecl-timeline2`... but now is too late I guess :'(